docs/en_US/Tutorial/QuickStart.rst

```diff
@@ -128,27 +138,29 @@ Three steps to start an experiment
 
 *Example:* :githublink:`mnist.py <examples/trials/mnist-pytorch/mnist.py>`
 
-**Step 3**\ : Define a ``config`` file in YAML which declares the ``path`` to the search space and trial files. It also gives other information such as the tuning algorithm, max trial number, and max duration arguments.
 
-.. code-block:: yaml
+Step 3: Config the Experiment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-   authorName: default
-   experimentName: example_mnist
-   trialConcurrency: 1
-   maxExecDuration: 1h
-   maxTrialNum: 10
-   trainingServicePlatform: local
-   # The path to Search Space
-   searchSpacePath: search_space.json
-   useAnnotation: false
-   tuner:
-     builtinTunerName: TPE
-   # The path and the running command of trial
-   trial:
-     command: python3 mnist.py
-     codeDir: .
-     gpuNum: 0
+Config the experiment in the YAML file which declares the ``path`` to the search space and trial files. It also gives other information such as the tuning algorithm, max trial number, and max duration arguments.
 
+.. code-block:: yaml
+
+   experimentName: MNIST               # An optional name to distinguish the experiments
+   searchSpaceFile: search_space.json  # The path to the search space file
+   trialCommand: python3 mnist.py      # NOTE: change "python3" to "python" if you are using Windows
+   trialCodeDirectory: .               # The path to the trial code
```

Comment:
As a template, I think it's save to remove this line and ask the user to put config file in trial code's top directory.

Reply:
okay, will update it